src/components/PageTitle.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+import { Title } from "./PageTitleStyled";
+
+const PageTitle = (props) => {
```

Comment:
as I mentioned above it would be nice to destructure prop here

Reply:
Done